traveler-swift-core/TravelerKit/Models/OffsetSeconds.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  OffsetSeconds.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-07-18.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+/// Contains information about UTC offset seconds
+public struct OffsetSeconds {
```

Comment:
Fight the api team to do the conversion to Double on their end (or int if its in minutes)

This will save work on two platforms and be less error prone